src/CMakeLists.txt

```diff
@@ -117,6 +117,16 @@ elseif(${PLATFORM} MATCHES "Raspberry Pi")
   set(PLATFORM_CPP "PLATFORM_RPI")
   set(GRAPHICS "GRAPHICS_API_OPENGL_ES2")
 
+  add_definitions(-D_GNU_SOURCE=1)
+  
+  find_library(OPENGL_LIBRARY OpenGL)
+  find_library(GLESV2 brcmGLESv2 HINTS /opt/vc/lib)
```

Comment:
Is `/opt/vc` some standard path for the VideoCore? Who installs this?

Reply:
I think it comes with every installation of raspbian. I don't know about other distibutions.
I got the paths from the Makefile https://github.com/raysan5/raylib/blob/e65d15f6ad6f9ade05999b66fdd91d3f6eb2a2d0/src/Makefile#L388

Reply:
ok